index.html

```diff
@@ -18,140 +18,174 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header>
-      <h1>YEONSINSA</h1>
-      <nav>
-        <button><i class="fa-solid fa-bars"></i></button>
+    <header class="header">
+      <h1 class="header__title">YEONSINSA</h1>
+      <nav class="header__nav">
+        <button class="header__button"><i class="fa-solid fa-bars"></i></button>
       </nav>
     </header>
 
     <main>
-      <nav>
-        <h2>CATEGORY</h2>
-        <ul>
-          <li>ALL</li>
-          <li>CLOTHES</li>
-          <li>SHOES</li>
-          <li>ETC</li>
-        </ul>
+      <nav class="nav">
+        <h2 class="nav__title">CATEGORY</h2>
+        <section class="nav_content">
+          <article class="nav__category">
+            <input type="checkbox" class="nav__checkbox" />
```

Comment:
bem방법론 조금더 디테일하게 하면 사실 여기서부터는 input이 article태그의 자식이니까 nav 밑의 블록으로 표현하기보다는
```suggestion
<input type="checkbox" class="nav__category__checkBox"/>
```
가 조콤더 명확하게 보이겠찌?!

Reply:
[bem 방법론] (https://velog.io/@yesdoing/BEM-Block-Element-Modifier-Quick-start)은 요기 링크 참고했는데 
여기서 계층 구조로 선언하는 것은 추천하지 않아서 이렇게 이름 지었던거여써! 어떤게 더 적절한 방법일까 고민되네 💭..!